fontscan/langset.go

```diff
@@ -0,0 +1,22 @@
+package fontscan
+
+// LanguageID is a compact representation of a language
+// this package has orthographic knowledge of.
+type LanguageID uint16
```

Comment:
If I'm reading the code right, we're only using 9 bits of the 16. That's probably worth documenting somewhere. Maybe on the `langset` itself, since it's talking about the "high bits" being bits 8-6.